docs/tutorials/use.md

```diff
@@ -76,12 +38,12 @@ REPOSITORY                                  TAG                   IMAGE ID
 localhost/podenv/firefox                    latest                713d6fa70a9f   46 seconds ago   556 MB
 ```
```

Comment:
It would be handy to have a podenv images command as a wrapper on podenv generated images as well with a janitor option.

Reply:
Indeed, I added a note to the roadmap.